application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -156,7 +149,8 @@ private void sendCooldownResponse(SlashCommandInteractionEvent event) {
             logger.warn("Can't find the last help thread created by the user with id ({})",
                     user.getId());
             event
-                .reply("Sorry, something went wrong try again after %s".formatted(cooldownDuration))
+                .reply("Sorry, something went wrong. Please try again after %s"
```

Comment:
is this missing a dot at the end of the sentence?

Reply:
done